project/chrome-extension/information-display.js

```diff
@@ -1,35 +1,86 @@
 //Calls the showInfo method after the whole document loads.
-window.onload = showInfo;
+window.onload = init;
 
-//Shows the information about the word that was choosen.
-async function showInfo(){
-
-  wordFoundAndIntro();
-  createDescription();
-  addSources();
+//Shows the word that was checked and a brief intro.
+function wordFoundAndIntro(word) {
+  //Shows the word that was clicked by the user
+  const wordFound = document.getElementById("wordFound");
+  //Adds the word of the id to the wordFound element
+  wordFound.innerText =
+    "Word to be checked: " + word;
+  //gets the introduction element
+  const intro = document.getElementById("wordTitle");
+  //Adds the word of the id to the wordTitle element
+  intro.innerText =
+    "See the latest information about " + word;
 }
 
-//Shows the word that was checked and a brief intro.
-function wordFoundAndIntro(){
-   //Shows the word that was clicked by the user
-   const wordFound = document.getElementById("wordFound");
-   //Adds the word of the id to the wordFound element
-   wordFound.innerText = "Word to be checked: " + localStorage.getItem("rdoIdVarKey");
-   //gets the introduction element
-   const intro = document.getElementById("wordTitle");
-    //Adds the word of the id to the wordTitle element
-   intro.innerText = "See the latest information about " + localStorage.getItem("rdoIdVarKey");
+//Finds the picture of the word on the dictionary.
+function addPicture(imageURL) {
+
+      var img = document.createElement("img");
+      img.src = imageURL;
+      var src = document.getElementById("imageAdded");
+      src.appendChild(img);
+  
 }
 
 // Add information aboout word to info.html
-function createDescription(){
-  const description = document.getElementbyId("description");
+function createDescription(description) {
   // TODO(@SamRod33 or @SofiaPSU): Add to the description by using description.innerText
+      document.getElementById("description").innerText += description;
+  
 }
 
 // Add sources, such as link or references, to the information used for the
 // description of the word.
-function addSources(){
-  const source = document.getElementById("sources");
+function addSources(sourceURL) {
   // TODO(@SamRod33 or @SofiaPSU): Add to the sources by using source.innerText
+
+      var a = document.createElement('a');
+      a.setAttribute('href',sourceURL);
+      // a.innerHTML = "Source: "+topics[i].source.sourceName;
+      a.innerHTML = "Source: "+ sourceURL;
+      document.getElementById('sources').appendChild(a);
+
 }
+  function loadJSON(callback) {
+    var xObj = new XMLHttpRequest();
+    xObj.overrideMimeType("application/json");
+    xObj.open('GET', 'data.json', true);
```

Comment:
Json file is called words.json, I know data.json is the example file, but we should integrate into using words.json